Code-the-Dream-School/dd-prac-team1-front
# PRA-159 Filtering search results

##  Filtering search results

## Requirements

1) When you search for an item, you can filter the results.

2) If a tag consists of more 2 or more words, you can search through all of them.

## Notes

None

## Test Steps

 Search for a word and then filter the results with this word, then press "all categories" and check that you can filter through all the results again. Also if you have tags which consist of two words, try to search for the second one.

## UI/UX Outcome

<img width="1240" alt="Screen Shot 2023-08-23 at 12 28 16" src="https://github.com/Code-the-Dream-School/dd-prac-team1-front/assets/114008959/cc025e77-054c-488b-a3cc-415660a59b2f">

## Checklist

- The Linear ID is in the PR title
- My code follows best practices
- Documentation is included where needed
